test/vaadin-radio-group.html

```diff
@@ -613,14 +614,30 @@
     });
 
     describe('vaadin-radio-group wrapping', () => {
-      let vaadinRadioButtonGroup;
+      let vaadinRadioButtonGroup, vaadinRadioButtonList;
 
       beforeEach((done) => {
         vaadinRadioButtonGroup = fixture('wrapping');
+        vaadinRadioButtonList = vaadinRadioButtonGroup.querySelectorAll('vaadin-radio-button');
         vaadinRadioButtonGroup._observer.flush();
         animationFrameFlush(done);
       });
 
+      it('should value on the vaadin-radio-group is updated at the time when change is fired', () => {
+        vaadinRadioButtonGroup.addEventListener('change', e => {
+          expect(vaadinRadioButtonGroup.value).to.equal(vaadinRadioButtonList[2].value);
+        });
```

Comment:
Both tests that you added should use callback to make sure they wait for `change` event:

```js
it('should value on the vaadin-radio-group is updated at the time when change is fired', done => {
  vaadinRadioButtonGroup.addEventListener('change', e => {
    expect(vaadinRadioButtonGroup.value).to.equal(vaadinRadioButtonList[2].value);
    done();
  });
  vaadinRadioButtonList[2].click();
});
```

Using `done` callback is required whenever you wait for an event listener or any other async operation in test.

Reply:
added!